packages/in_app_purchase/example/lib/main.dart

```diff
@@ -48,7 +48,8 @@ class _MyAppState extends State<MyApp> {
   }
 
   Future<List<Widget>> buildStorefront() async {
-    final bool available = await InAppPurchaseConnection.instance.isAvailable();
+    final InAppPurchaseConnection connection = InAppPurchaseConnection.instance;
```

Comment:
This segment isn't going to work on Android yet. Could this file be reverted until a later patch where Android is also done and the unified `Product` is written?